ec2/InstanceType.dhall

```diff
@@ -1328,13 +1196,20 @@ let alternatives =
                     }
             }
 
-      in  { Parameters = Parameters, function = alternatives }
+      in  { Parameters, function = alternatives }
 
 let upsize
     : InstanceType → Optional InstanceType
-    =   λ(instanceType : InstanceType)
-      → merge
-          { c5-large = Some InstanceType.c5-xlarge
+    = λ(instanceType : InstanceType) →
+        merge
+          { t3-nano = Some InstanceType.t3-nano
+          , t3-micro = Some InstanceType.t3-micro
+          , t3-small = Some InstanceType.t3-small
+          , t3-medium = Some InstanceType.t3-medium
+          , t3-large = Some InstanceType.t3-large
+          , t3-xlarge = Some InstanceType.t3-xlarge
+          , t3-2xlarge = Some InstanceType.t3-2xlarge
```

Comment:
You didn't upsize anything here, you returned the same size?

Reply:
fixed

Reply:
Not fixed

Reply:
didn't push yet. wait for it! :)

Reply:
Please push then comment ;)